shallow_backup/utils.py

```diff
@@ -55,16 +55,15 @@ def mkdir_warn_overwrite(path):
 			sys.exit()
 	elif not os.path.exists(path):
 		os.makedirs(path)
-		print(Fore.BLUE + Style.BRIGHT + "CREATED DIR: " + Style.NORMAL + path + Style.RESET_ALL)
+		print_path_blue("CREATED DIR:", path)
 
 
 def destroy_backup_dir(backup_path):
 	"""
 	Deletes the backup directory and its content
 	"""
 	try:
-		# TODO: PRINT PATH STYLING. PATH SHOULD NOT BE BOLDED.
-		print_red_bold("Deleting backup directory: {}".format(backup_path))
+		print_path_red("Deleting backup directory:", backup_path)
 		rmtree(backup_path)
 	except OSError as e:
 		print("{} Error: {} - {}. {}".format(Fore.RED, e.filename, e.strerror, Style.RESET_ALL))
```

Comment:
Let's add `print_red_bold` error formatting here to standardize it with the rest of the codebase.

Reply:
Small doubt here: 
The intent is to keep the error and path in bold for this specific message?

Reply:
I meant that the path should *not* be bold.